azure-dataplane-tests/src/main/java/com/azure/ai/formrecognizer/documentanalysis/dpg/DocumentModelsClientBuilder.java

```diff
@@ -262,8 +262,16 @@ private FormRecognizerClientImpl buildInnerClient() {
         return client;
     }
 
+    @Generated
+    private void validateBuilder() {
+        // This method is invoked from 'createHttpPipeline' when preparing the HTTP pipeline for the new client.
+        // Developer can customize this method, to validate that the necessary conditions are met for the new client.
```

Comment:
Currently we will validate required fields not be null?

Reply:
In current (main) code, we won't.

In this PR, see if we can add the default validation code, for required property without default value -- e.g. (typical case) the endpoint.

Reply:
I mean this pr. Are we validating all the required property to be non-null, or only `endpoint` this time?

Reply:
To all "required"